pyunlocbox/solvers.py

```diff
@@ -54,7 +54,7 @@
 
 
 def solve(functions, x0, solver=None, atol=None, dtol=None, rtol=1e-3,
-          xtol=None, maxit=200, verbosity='LOW'):
+          xtol=None, maxit=200, verbosity='LOW', persistent=False):
```

Comment:
Is `persistent` a standard name for such kind of functionality?

Reply:
I think not, I went with `inplace` instead.

Reply:
I was going to suggest that name too. :)